src/apps/authentication/router.py

```diff
@@ -1,13 +1,8 @@
 from fastapi.routing import APIRouter
 
-from apps.authentication.api.auth import (
-    access_token_delete,
-    create_user,
-    get_access_token,
-)
+from apps.authentication.api.auth import delete_access_token, get_access_token
 
 router = APIRouter(prefix="/auth", tags=["Authentication"])
 
-router.post("/signup")(create_user)
-router.post("/access-token")(get_access_token)
-router.post("/signout")(access_token_delete)
+router.post("/token")(get_access_token)
```

Comment:
could you use general path naming, like login and logout and both will be post

Reply:
Actually if you remember we aligned it with frontend team :)

Reply:
okay in that case, because last time they have asked me why it is confusing